### ISSUE TITLE
"polymer build" fails because it can't differentiate between in-app and out-of-app relative paths

### ISSUE THREAD

**Issue by tekkamanendless** [Polymer/polymer-cli#738]
### Description
I'm trying to run `polmer build` on an existing application so that I can get it nicely bundled.  However, my UI and my API live on the same domain.  For example, the UI is at "https://www.example.com/ui", while the API is at "https://www.example.com/api".  In one place in the application, I have a JSONP script included via:
```
<script src="/api/v1/someEndpoint?jsonp=someCallback"></script>
```

The expectation here is that this is done very early on in the site-loading so that every further item has access to whatever `someCallback` does with that data.

When I run `polymer build`, it fails with:
```
info:    Clearing build/ directory...
info:    (unbundled) Building...
error:   Uncaught exception: Error: ENOENT: no such file or directory, open '/home/me/path/to/app/api/v1/someEndpoint'
error:   Error: ENOENT: no such file or directory, open '/home/me/path/to/app/api/v1/someEndpoint'
    at Error (native)
```

I have specifically only included `src/**/*.html` and `src/**/*.js` files in my `polymer.json` `sources` entry, but it still is failing due to this error.

I see why it's doing this, but the only workaround that I can think of right now would be to embed full URLs instead of absolute paths.  I would prefer not to embed full URLs into the code because the same code base is used on multiple domains; embedding the full URL would cause me a huge maintenance headache.

I also suppose that I can find some different way of loading some global data from the API early on for everything else to share, but I would prefer not to refactor a bunch of stuff just to make `polymer build` happy.

A simple list of paths to ignore would be sufficient to solve this problem.  For example, `"ignore": [ "/api/**/*" ]` would probably do the trick.

### Versions & Environment
- Polymer CLI: node0.18.3
- node: v6.10.3
- Operating System: Ubuntu 16.04

#### Steps to Reproduce
1. Create an application project: `polymer init application`
2. Update `polymer.json` to only include `src/**/*.html` and `src/**/*.js` files.
2. Add script tag to index.html: `<script src="/api/v1/someEndpoint"></script>`
3. Build: `polymer build`

#### Expected Results
Successful build.

#### Actual Results
Error about not being able to find "/api/v1/someEndpoint".

**Comment by FredKSchott** [comment]
Agreed, this is something that we don't currently support that we should.
